src/handler.cpp

```diff
@@ -1488,6 +1491,8 @@ void char_to_room(struct char_data * ch, struct room_data *room)
   _char_with_spell_to_room(ch, SPELL_SHADOW, ch->in_room->shadow);
   _char_with_spell_to_room(ch, SPELL_LIGHT, ch->in_room->light);
   _char_with_spell_to_room(ch, SPELL_POLTERGEIST, ch->in_room->poltergeist);
+  
+  SendGMCPRoomInfo(ch, ch->in_room);
```

Comment:
This works, but folks are going to see odd results. We sometimes change character room locations during processing of other code before restoring them to where they were. This is invisible to players, but will become visible with this change.

Reply:
Maybe we should only add it on the do_move code? Hmm